src/HypothesisClient/ApiClient/ApiClient.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace eLife\HypothesisClient\ApiClient;
+
+use eLife\HypothesisClient\Credentials\Credentials;
+
+interface ApiClient
+{
+    public function getCredentials() : Credentials;
```

Comment:
Is this needed? Never seems to be used externally?

Reply:
No. Will remove.